internal/bulk_test.go

```diff
@@ -18,6 +18,7 @@ import (
 func TestBulk(t *testing.T) {
```

Comment:
I took a look at the tests of the project and I didn't manage to figure out the testing approach fully. Not sure why some things are tested in some places and some other things are not. It was hard for me to find clear patterns to be consistent with. So, I finally opted for adding these two basic cases. They don't cover all the code I wrote, but I don't think the test suite were aiming for 100% coverage.

Anyways, any suggestions about how better test my changes will be more than welcome.

Reply:
Testing is a bit random (probably my influence) and there is no "one-way". Most tend to add tests in whatever way they are confortable with. I'm not a huge fan of the output based testing for example, and tables can be confusing, but some people like them and as long as there is something that works, then it's probably okay :-)